src/onnx/parse_batchnorm.cpp

```diff
@@ -37,61 +36,15 @@ struct parse_batchnorm : op_parser<parse_batchnorm>
     instruction_ref parse(const op_desc& /*opd*/,
                           const onnx_parser& parser,
                           const onnx_parser::node_info& info,
-                          std::vector<instruction_ref> args) const
+                          const std::vector<instruction_ref>& args) const
     {
         float epsilon = 1e-5f;
```

Comment:
The default attribute value is set in parse_... and in the op builder. This might lead to some unintuitive behavior should someone try to change it in the op builder. Probably better to have a conditional here and use the default in the op builder.